uiowa/uiowa
# Admissions - Student Profiles Grid Filtering Fix

Resolves #4528

During the ListBlock refactors, the ability to filter student profiles on this Admissions block was lost. Re-saved the view config and exported.

## To Test

- `blt ds --site=admissions.uiowa.edu`
- IMO, stage file proxy for these grid images keeps the display from breaking
- As a web editor try placing the student profiles grid block. There should be filtering options. Try a couple.
- The template for a student profile should not be set to filter any of these filters (show all)
